lib/resources/example_json/r4_examples_device.rb

```diff
@@ -0,0 +1,338 @@
+module Cerner
+    module Resources
+  
+      R4_DEVICE ||={
```

Comment:
The spacing for this JSON seems off. Should review Allergy's to see proper spacing

Reply:
Fixed the formatting [here](https://github.com/cerner/fhir.cerner.com/pull/286/commits/4f208c84cb5b9f4f6973dd9983f9d4504d8d001a#diff-a1a14d90f1de88415531f8370cae832fR5)

Reply:
+1